package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@tram-one/tram-logo",
+  "version": "4.0.0",
+  "description": "This logo was created for the Tram-One Project, designed by Daniel Jurman",
+  "main": "logo.svg",
+  "files": [
+    "dist",
+    "variants",
+    "logo.svg"
+  ],
+  "scripts": {
+    "serve": "serve variants -p 4455",
+    "prebuild": "rm -rf dist && mkdir -p dist",
```

Comment:
only works on unix, this package shouldn't get that much foot traffic, and we when do want to make this more cross-platform, that should be an easy fix.